recipes/django-rest-hooks/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.2.2" %}
+
+package:
+  name: django-rest-hooks
+  version: {{ version }}
+
+source:
+  fn: django-rest-hooks-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/django-rest-hooks/django-rest-hooks-{{ version }}.tar.gz
+  sha256: c121ffbc1ee4a116bd9d5b39e3c97152f409d84dd4d13e121465d20a03ac0dcf
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - django >=1.4
+    - requests
+
+  run:
+    - python
+    - django >=1.4
+    - requests
+
+test:
+  imports:
+    - rest_hooks
+
+about:
+  home: http://github.com/zapier/django-rest-hooks
+  license: BSD License
```

Comment:
We can merge this before they answer BTW as this is "correct" according to the repository. But we will need to fix the feedstock once they answer. So it is up to you if you want to wait or merge.